Tweak.xm

```diff
@@ -209,6 +210,15 @@ static CGFloat offset = 0;
 	return %orig(packageName, [NSBundle bundleWithPath:@"/Library/Application Support/ProudLock2"]);
 }
 %end	// %hook BSUICAPackageView
+
+%hook CSCombinedListViewController
+- (UIEdgeInsets)_listViewDefaultContentInsets {
+    UIEdgeInsets orig = %orig;
+
+    orig.top += offset;
+    return orig;
+}
+%end	// %hook CSCombinedListViewController
```

Comment:
_Note:_ ```%hook```s inside ```%group```s are generally commented with three slashes in my code, as it keeps things a little more structured in my opinion. Again, this isn't necessary whatsoever, but I think it's nicer to keep a consistent code style.